content/coronavirus_landing_page.yml

```diff
@@ -126,6 +126,8 @@ content:
           list:
            - label: Planning inspections
              url: /guidance/coronavirus-covid-19-planning-inspectorate-guidance
+           - label: Renting: guidance for landlords, tenants and local authorities
```

Comment:
This is causing an error because colons can't be used in values.  If you change the line to

```
           - label: "Renting: guidance for landlords, tenants and local authorities"
```

It will work and I can merge this.

Reply:
@barrucadu how do I edit it again?

Reply:
If you look at the version of the file on your branch https://github.com/alphagov/govuk-coronavirus-content/blob/helengds-patch-1/content/coronavirus_landing_page.yml you can edit it in the same way again, with the pencil button at the top right of the file.